_posts/2020-08-03-working.md

```diff
@@ -7,6 +7,95 @@ categories: Status
 
 This report will be a little bit longer than normal cause it will comprehend the last week changes too. So let's start.
 
-  1. Status:
-  2. Abstract:
-  3. Next:
+  1. Status: We have been accompanying the status and behavior of the `update_versions`alterations on 
+  [`Cirlce`](https://circleci.com/) to check if any further modifications or bug removals would be necessary, with some
```

Comment:
you most likely want to use the real circle url, rather than just the generic url

Reply:
Can I use this one instead ? https://app.circleci.com/pipelines/github/regro/circle_worker/20894/workflows/8ab11518-fbab-42ff-b2b4-15e3953592f2/jobs/39240

Reply:
I would use the pipelines link https://app.circleci.com/pipelines/github/regro/circle_worker?branch=master

Reply:
Thanks